Feature/car 3 create sql scripts

mironuk opened the pull request.

mironuk committed:
CAR-3 : Create SQL scripts

- Added DB schema.
- Fixed DB entities to correspond schema.
- Fixed test to correspond entities.
- Created initial create.sql with TODO.

mironuk committed:
CAR-3 : Create SQL scripts

- Fixed DB schema letter case.
- Fixed DB entities - added @NotNull to @ManyToOne dependencies.
- Finished create.sql.

mironuk committed:
CAR-3 : Create SQL scripts

- Fixed DB schema - removed engine and default charset.
- Fixed create.sql - added indentations.

majagabajaga commented on src/main/java/carservice/web/data/model/User.java:
```diff
@@ -17,13 +17,19 @@
 	@Column(name="user_id")
 	private long userId;
 
+    @Column(name="email")
```
Make unique and not null.

mironuk replied:
Done.

mironuk committed:
CAR-3 : Create SQL scripts

- Fixed DB schema - BIGINT replaced to INT UNSIGNED for all IDs.
- Fixed DB schema and User entity - made email unique and not null.
- Renamed create.sql to create-database.sql.
- Renamed schema.sql to create-schema.sql.

mironuk committed:
CAR-3 : Create SQL scripts

- Fixed DB schema - replaced UNSIGNED INT to INT UNSIGNED.

mironuk committed:
CAR-3 : Create SQL scripts

- Fixed DB schema - replaced pass to password in user table.
